src/screens/CameraScreen.js

```diff
@@ -1,26 +1,296 @@
-import React from "react";
-import { View, Text, StyleSheet, Button } from "react-native";
-
-const CameraScreen = (props) => {
-  return (
-    <View style={styles.screen}>
-      <Text>Camera Screen</Text>
-      <Button
-        title="Posting"
-        onPress={() => {
-          props.navigation.navigate({ routeName: "Posting" });
-        }}
-      />
-    </View>
-  );
-};
+import React, { Component } from "react";
+import { View, Text, SafeAreaView, StatusBar, StyleSheet } from "react-native";
+import { MaterialIcons } from "@expo/vector-icons";
+import { Camera } from "expo-camera";
+import * as Permissions from "expo-permissions";
```

Comment:
Is this star import necessary here? I suggest we shy away from star imports whenever we can.

Reply:
I dont think its possible to remove star import. Its required to do that way. Please check [Permissions](https://docs.expo.io/versions/v38.0.0/sdk/permissions/) docs

Reply:
I wonder if you can replace * for specific types. Check [docs](https://docs.expo.io/versions/v38.0.0/sdk/permissions/#usage-in-bare-workflow) out.